build/depends.py

```diff
@@ -275,9 +265,9 @@ def configure(self, build, conf):
         else:
             build.env.Append(CPPDEFINES='QT_SHARED')
 
-        if qt5:
-            # Enable qt4 support.
-            build.env.Append(CPPDEFINES='QT_DISABLE_DEPRECATED_BEFORE')
+        # Enable qt4 deprecated methods.
+        # TODO(rryan): Remove this.
+        build.env.Append(CPPDEFINES='QT_DISABLE_DEPRECATED_BEFORE')
```

Comment:
Can this be removed now?

Reply:
Nope, it requires some changes in Mixxx to remove which I started in #1806, but need to take care of @daschuer's comments.